src/main/kotlin/com/kotlin/boardproject/global/filter/TokenAuthenticationFilter.kt

```diff
@@ -1,15 +1,16 @@
-package com.kotlin.boardproject.common.filter
+package com.kotlin.boardproject.global.filter
 
-import com.kotlin.boardproject.common.util.getAccessToken
-import com.kotlin.boardproject.common.util.log
-import org.springframework.security.core.context.SecurityContextHolder
-import org.springframework.web.filter.OncePerRequestFilter
+import com.kotlin.boardproject.global.util.AuthTokenProvider
```

Comment:
🚫 **[ktlint]** <standard:import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end